adafruit_ina219.py

```diff
@@ -33,14 +33,12 @@
 from micropython import const
 
 try:
-    import typing
+    from typing import TYPE_CHECKING
 
-    from busio import I2C
+    if TYPE_CHECKING:
```

Comment:
I think I'd rather not add this explicit check for `TYPE_CHECKING`. 

It's more pythonic and correct as you noted in the comment on the other PR. But since we already guard the typing imports with try/catch, and we don't have the `TYPE_CHECKING` concept in CircuitPython it doesn't really accomplish anything other than making the libraries slightly larger which can be valuable flash storage space on some of the more constrained hardware that CircuitPython supports.